### ISSUE TITLE
Accessibility Issue with Hamburger Menu in Usability Theme?

### ISSUE THREAD

**Issue by DHaslip** [wet-boew/wet-boew#8231]
In the latest version of the usability theme, the close "x" in the hamburger menu doesn't have enough contrast to meet WCAG, according to our investigation.  Has anyone else addressed this?

**Comment by DHaslip** [comment]
Has anyone had a chance to look at this?

**Comment by duboisp** [comment]
@DHaslip what color are you suggesting for the contrast?

I think that there is a technical specification at tbs defining exactly how the GC web usability theme should be styled. 

The next step will be:
1. Find the GCWU specification documents and see it specify a color for the close button "X".
2. If there is not color, what color are you proposing that would enough contrast?
3. If there is a color, is that color have enough contrast? In the negative, what are you proposing?
4. Submit your proposal/find here.

After all those check/proposal, the next step will be to submit a PR in the GCWU theme.
